apps/functions/applications-migration/common/migrators/exam-timetable-migration.js

```diff
@@ -95,23 +97,18 @@ const mapTimetableFromItems = (caseReference, timetableItems) => {
 	return timetableItems.reduce(
 		(
 			timetable,
-			{
-				eventID,
-				eventType,
-				eventTitle,
-				eventDate,
-				eventDeadlineStartDate,
-				eventLineItemDescription
-			}
+			{ eventId, type, eventTitle, date, eventDeadlineStartDate, description, eventLineItems }
 		) => {
 			timetable.events.push(
 				removeNullValues({
-					eventId: Number(eventID),
-					type: eventType,
+					eventId: Number(eventId),
+					type,
 					eventTitle,
-					description: eventLineItemDescription,
-					date: eventDate,
-					eventDeadlineStartDate
+					description,
+					date,
+					eventDeadlineStartDate,
+					// seem to be all null in ODW and we need an array to pass validation
+					eventLineItems: eventLineItems?.length > 0 ? eventLineItems : []
```

Comment:
Given that ODW do not have lineItem data from Horizon, I am wondering whether we should extract it ourselves from the description field passed to us?  In CBOS we use this function to make the lineItem data from the description:  extractDescriptionAndLineItems